dev/Sidrah-Madiha/Importance score for dataset training samples/helper_for_senstivity_calculation.py

```diff
@@ -0,0 +1,57 @@
+from sklearn import metrics
+import numpy as np
+def model_performance_evaluater_score (model_func,X_train, y_train, X_test, y_test):
+    '''returns accuracy of a model when training and testing data and target labels are provided
+    inputs:
+    model_func : model for fitting training data
+    X_train : training data for model fitting
+    y_train : training data's target labels
+    X_test : testing data for model's accuracy calculation
+    y_test: testing data's target labels for model's accuracy calculation'''
+    model_func.fit(X_train, y_train)
+    # y_pred=model_func.predict(X_test) 
+    # return metrics.accuracy_score(y_test, y_pred)
+    # I am not sure if accuracy or score is record for senstivity calculation so I commented out accuracy code
```

Comment:
I'm pretty sure these do the same think. I thing `model_func.score()` is just a shortcut.